frontend/add-leads/add-edit-materialEntry.component.tsx

```diff
@@ -0,0 +1,70 @@
+import { useCss } from "kremling";
+import React from "react";
+import css from "./add-edit-materialEntry.css";
+
+export default function MaterialEntry({
+  materialProp,
+  handleDelete,
+  handleEdit,
+}) {
+  const [editing, setEditing] = React.useState(false);
+  const [material, setMaterial] = React.useState({ id: "", name: "" });
+
+  React.useEffect(() => {
+    setMaterial(materialProp);
+  }, []);
+
+  function OnValueChange(evt) {
+    material.name = evt.target.value;
+    setMaterial(material);
+  }
+
+  function onSave() {
+    handleEdit(material);
+    setEditing(!editing);
+  }
+
+  return (
+    <div key={material.id} {...useCss(css)} className="container">
```

Comment:
The key here is not needed, since it's not part of an array. https://reactjs.org/docs/lists-and-keys.html

```suggestion
    <div {...useCss(css)} className="container">
```